versions.props

```diff
@@ -12,7 +12,7 @@ com.google.auto.service:auto-service-annotations = 1.0
 com.google.auto:auto-common = 1.1
 com.google.code.findbugs:findbugs-annotations = 3.0.1
 com.google.code.findbugs:jsr305 = 3.0.2
-com.google.dagger:* = 2.37
+com.google.dagger:* = 2.38
```

Comment:
This is the one that seems to break tests.

Reply:
This'll be why: 
![image](https://user-images.githubusercontent.com/13332324/126661319-4c4682ac-566c-4409-82cf-44dc3fb129d8.png)

https://github.com/google/dagger/releases/tag/dagger-2.38